src/platform/plugins/shared/discover/public/application/main/state_management/redux/actions/initialize_session.ts

```diff
@@ -113,7 +113,7 @@ export const initializeSession: InternalStateThunkActionCreator<
     if (TABS_ENABLED && !wasTabInitialized) {
       const tabInitialGlobalState = tabState.initialGlobalState;
 
-      if (tabInitialGlobalState?.filters) {
+      if (tabInitialGlobalState?.filters?.length) {
```

Comment:
Just curious what prompted this change?

Reply:
I had an optimization in mind but reverted now https://github.com/elastic/kibana/pull/225622/commits/ec3427b5dca00ec9974d344c8d07473728c65d10